boards/opentitan/README.md

```diff
@@ -217,14 +217,45 @@ $ make flash-app APP=<...> OPENTITAN_TREE=/home/opentitan/
 You will need to have the GCC version of [RISC-V 32-bit objcopy](https://github.com/riscv-collab/riscv-gnu-toolchain/blob/master/README.md) installed as
 the LLVM one doesn't support updating sections.
 
-### Programming Apps in Verilator
+### <u> Programming Apps in Verilator</u>
 
-An app can be bundled and loaded with the kernel into Verilator with:
+A **single app** in `.tbf (tock binary format)` can be bundled and loaded with the kernel into Verilator with:
 
 ```shell
 $ APP=<...> make BOARD_CONFIGURATION=sim_verilator verilator
 ```
 
+### <u> Libtock-C App Verilator Example</u>
+
+To load a libtock-c app, we can do the following to load the `c_hello` sample app:
+
+<u>**Build app:**</u>
+```
+$ git clone https://github.com/tock/libtock-c.git
+$ cd libtock-c/examples/c_hello/
+$ make RISCV=1
+```
+<u>**Load and Run:**</u>
+
+Now, in the Opentitan board directory in tock (`tock/boards/opentitan/earlgrey-cw310`)
+```
+$ APP=/home/twilfred/wdc/libtock-c/examples/c_hello/build/rv32imc/rv32imc.0x20030080.0x10005000.tbf make BOARD_CONFIGURATION=sim_verilator verilator
+```
+pay attention to the flash/memory mapping of the tbf  file and ensure that it falls within the expected ranges in `layout.ld`.
```

Comment:
This comment doesn't seem to match the rest of this change. If someone is reading to learn how to clone the libtock-c repo and compile a libtock app, are they going to be able to check the flash and memory mapping? I suspect they wouldn't know what that means. I think either remove this line, or specify which TBF they need (it shouldn't change unless the kernel, and this readme, changes), or add some explanation.

Reply:
removed HTML (for underlines), and updated the README to specify which `tbf` to use.